Microsoft.Toolkit.Uwp.UI.Animations/Effects/AnimationEffect.cs

```diff
@@ -141,13 +144,13 @@ public abstract class AnimationEffect
                 foreach (var effectProperty in _effectProperties)
                 {
                     var animation = Compositor.CreateScalarKeyFrameAnimation();
-                    if (easingType == EasingType.Default)
+                    if (easingType == EasingType.Default && easingMode == EasingMode.EaseOut)
```

Comment:
That's the only thing I want to clear. Aside from that, PR approved.

Reply:
ping @JohnnyWestlake, code freeze is Wednesday, could you address this before then?

Reply:
From what I can see (and @nmetulev can confirm?) I think the original intention was to use the default composition easing when we use the default easing type (If you don't specific an easing function for a composition keyframe it eases by default with some internal ease). Now that we've got easing modes this actually eases closest to Cubic Ease Out - however cubic ease out doesn't actually ease like teh default composition function who keyspline is much closer to `(0.395, 0.56, 0.06, 0.95)` rather than `(0.215, 0.610, 0.355, 1)` used by Cubic Ease Out. With that in mind I have two choices: 

1 - Don't use composition default easing and explictly use cubic ease out
2 - Use composition default for composition, and for storyboards use a keyspline that matches it.

For brevity I've gone with option 1